src/components/Modal/Modal.tsx

```diff
@@ -34,42 +32,41 @@ const Header = styled.div`
 `;
 
 const Content = styled.div`
-    overflow: auto;
+    overflow-y: auto;
+    overflow-wrap: break-word;
     padding: 1em 1.5em 1.5em;
     width: 100%;
+    color: ${(props) => props.theme.main.foreground[2]};
+    background-color: ${(props) => props.theme.main.background[2]};
 `;
 
-const CloseButton = styled.button.attrs({
-    "aria-label": "Close Button",
-})`
+const CloseButton = styled.button`
     border: 0;
     border-radius: 50%;
+    background-color: ${(props) => props.theme.rainbow.lightGrey.background};
 
-    // TODO VFB-16 There are two options for the CloseButton styling. Either using a series of grey tones, or colours that match the theme of the modal
-
-    // TODO VFB-16 Either change to 'grey' or same colour as Header background
-    background-color: ${(props) => props.theme.secondaryBackgroundColor};
-
-    // TODO VFB-16 Ensure that this colour matches the header font colour
-    color: ${(props) => props.theme.secondaryForegroundColor};
+    path {
+        fill: ${(props) => props.theme.rainbow.darkGrey.background};
```

Comment:
shame we can't do something like this for moon/sun

Reply:
ye :((